data/json/recipes/tools/tools_hand.json

```diff
@@ -160,8 +161,8 @@
       [ "textbook_weparabic", 5 ]
     ],
     "using": [ [ "forging_standard", 3 ], [ "steel_tiny", 3 ] ],
-    "qualities": [ { "id": "ANVIL", "level": 3 }, { "id": "HAMMER", "level": 3 }, { "id": "GRIND", "level": 2 } ],
-    "//": "Specifically avoids blacksmithing requirement groups so that it's craftable without the metal fileset - this process would still be possible, just realistically take slightly longer.",
+    "qualities": [ { "id": "ANVIL", "level": 2 }, { "id": "HAMMER", "level": 3 }, { "id": "GRIND", "level": 2 } ],
+    "//": "Specifically avoids blacksmithing requirement groups so that it's craftable without the metal fileset - this process would still be possible, just realistically take slightly longer. Also avoids anvil 3 requirement since it is a direct requirement of making an proper anvil",
```

Comment:
I didn't write the first half of that note, someone else did, I just added the anvil part. I can clarify, however. Making a chisel is just hammer down a bar of steel into a flat tip, then grinding the flat tip until it's sharp. It would need need anything but a flat durable surface such as that provided by anvil 2, no curving, no notching.

Reply:
Updated the note with details.